### PR TITLE
Add dummy app for smoke test

### PR DESCRIPTION
**Pull request by gordonkristan**
This adds a dummy app for testing purposes. I added one really simple smoke test. It's really just there to assure that the ES6 code compiles without errors and that it generates valid ES5 code.

@stefanpenner Would you mind looking at this? This is my first real dive into Ember-CLI addons and I want to make sure I'm following good practice. Also, if you have any suggestions for other tests you'd like to see, just let me know.

### CODE REVIEW

File: tests/acceptance/simple-test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  afterEach: function() {
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('value of input', function(assert) {
+  visit('/');
+
+  andThen(function() {
```
Comment (stefanpenner) [inline tests/acceptance/simple-test.js]: also adding some es6 to the test tree would be great

`beforeEach() { }`
` andThen( () => {  }`

Comment (gordonkristan) [inline tests/acceptance/simple-test.js]: I didn't know if the tests were transpiled too. I'll go back and do that.

**Comment by stefanpenner** [comment]
LGTM once we get the test trees working.

**Comment by gordonkristan** [comment]
Sorry, I had a little snafoo with the Github app. (Is it just me or has it been getting worse lately?) Should be ready to merge now. Just one more question: do you see any unneeded Bower or NPM dependencies that I could remove?

**Comment by stefanpenner** [comment]
> Sorry, I had a little snafoo with the Github app. (Is it just me or has it been getting worse lately?)

no idea, i live in tmux + vim + git cli land

**Comment by stefanpenner** [comment]
we need to add travis

**Pull request merged**